services/tenant-ui/config/local.json

```diff
@@ -4,6 +4,6 @@
     "version": "latest"
```

Comment:
can we just ignore this file please? everyone seems to want something different...

Reply:
I thought I added it to gitignore? It should be gitignored and we shouldn't be checking it in

Reply:
Yeah, I was thinking the prompt for "are you sure" before deletion was a separate ticket.